tests/lib/onoff/src/main.c

```diff
@@ -270,7 +277,7 @@ static void setup_error(void)
 	reset_cli();
 }
 
-static void test_manager_init(void)
+ZTEST(test_onoff, test_manager_init)
```

Comment:
This commit is too hard to read; can you split it so the API update is done separately from the functional changes?